web/includes/functions.php

```diff
@@ -1668,6 +1668,10 @@ function getLoad()
 function getDiskPercent()
 {
     $total = disk_total_space(ZM_DIR_EVENTS);
+    if ( ! $total ) {
+        Error("disk_total_space returned false for " . ZM_DIR_EVENTS );
+        return 0;
```

Comment:
I'm thinking about the value we should return if disk_total_space fails:
- If we return 0, then the purge filter will not trigger and zoneminder will keep recording until writes fail due to disk full. I believe Zoneminder will simply stop running when this happens with a failed to write to disk message.
- if we return 100, then that will cause the purge filter to continually execute because it incorrectly thinks the disk is full. All events will eventually get deleted.
- if we return -1 to signal there was an error, then perhaps we can write additional code to handle that response and/or alert the user.

Thoughts?
Should we the third option?